solutions/fully-configurable/main.tf

```diff
@@ -2,7 +2,8 @@
 # Resource Group
 ########################################################################################################################
 locals {
-  prefix = var.prefix != null ? trimspace(var.prefix) != "" ? "${var.prefix}-" : "" : ""
+  prefix                        = var.prefix != null ? trimspace(var.prefix) != "" ? "${var.prefix}-" : "" : ""
+  effective_provider_visibility = var.provider_visibility != null ? var.provider_visibility : (var.region == "ca-mon" ? "vpe" : "private")
```

Comment:
can we add the local to provider.tf so it doesn't clog up main.tf code?

Reply:
updated logic to use just `private_endpoint_type`, not `visibility`.